src/components/modal/modal.tsx

```diff
@@ -119,10 +119,10 @@ export class Modal
   /** Specifies the size of the component. */
   @Prop({ reflect: true }) scale: Scale = "m";
 
-  /** Specifies the width of the component. Can use scale sizes or pass a number (displays in pixels). */
-  @Prop({ reflect: true }) width: Scale | number = "m";
+  /** Specifies the width of the component. */
+  @Prop({ reflect: true }) width: Scale = "m";
```

Comment:
Would this be considered a breaking change?

Reply:
Yeah we should probably consider it as one. Need to clean up a few things on this, will update with breaking change when ready.

Reply:
Yep, will update.

Reply:
Does `scale` not change the width of the component? Having css vars for width/height, and then also having props for width/scale seems confusing. If the `scale` prop doesn't currently change the width in this PR, maybe we should update it to do so, since it accepts the same type as the `width` prop anyway? Then we can just remove the width property since we have the CSS var. @SkyeSeitz @ashetland thoughts?

Reply:
Scale only changes the padding / text size. Width independently sets the modal to one of three default widths. Previously you could use the `| number` to pass a px value, or set directly on the modal, but the styles were moved inside the shadow root so the css vars replicate that.

We have a handful of components that have default widths that are able to be overridden - shell-panel, alert, etc., and I opened an issue about some of their inconsistent naming: https://github.com/Esri/calcite-components/issues/6172

Reply:
Based on the issue you logged shoud this prop be renamed widthScale to be consistent with the other components?

Reply:
Perhaps as a result of a more comprehensive audit, but short term I don't think we want to make a change. I still think it could just be a single default value that is only changed by css var. The linked issue is valid but post 1.0 since it would affect many other components.

Reply:
I agree that the `width = s | m | l` prop doesn't seem to make much sense with this change. A default width that is customizable via the css var would seem fairly straightforward.

Reply:
Let's try and tackle that change along with the other two issues - Modal has upcoming change after 1.0 with the proposed `fullscreen` changes and possibly removing `docked` when we add a `sheet`